com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -624,10 +624,11 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 depthDescriptor.msaaSamples = 1;// Depth-Only pass don't use MSAA
                 RenderingUtils.ReAllocateIfNeeded(ref m_DepthTexture, depthDescriptor, FilterMode.Point, wrapMode: TextureWrapMode.Clamp, name: "_CameraDepthTexture");
 
-                CommandBuffer cmd = CommandBufferPool.Get();
+                CommandBuffer cmd = renderingData.commandBuffer; //CommandBufferPool.Get();
                 cmd.SetGlobalTexture(m_DepthTexture.name, m_DepthTexture.nameID);
-                context.ExecuteCommandBuffer(cmd);
-                CommandBufferPool.Release(cmd);
+                context.ExecuteCommandBuffer(cmd); // do we need this here?
+                cmd.Clear(); // do we need this here?
```

Comment:
Not entirely sure why it's done here, I would assume it's only needed if there's any ScriptableRenderContext call that would break execution of command buffer in those passes. 
 
 Maybe it's a safer approach to avoid changing the execute / clear logic of command buffers until the RendererList PR lands?